### PR TITLE
Feature/4 jenkins setup

### PR DESCRIPTION
**Pull request by blabadi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,12 +11,13 @@
     
     <groupId>bio.overture</groupId>
     <artifactId>maestro</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <!-- from .mvn/maven.config -->
```
[the remark this answers is not in the document]
Comment (blabadi) [reply pom.xml]: > Question about all the `@Import` annotations.
> Could we not have just gotten away with relying on the component scanning?

I feel scanning is too much magic, harder to trace the dependency tree and reason about. 
been bit of it a lot before because it also scans test classes if they are in the same package, and you end up with duplicate beans all over the place.
